plugins/modules/route53.py

```diff
@@ -166,6 +190,11 @@
       returned: always
       type: str
       sample: PRIMARY
+    geo_location:
+      description: geograpic location based on which Route53 resonds to DNS queries.
+      returned: when configured
+      type: dict
+      sample: { continent_code: "NA", country_code: "US", subdivision_code: "CA" }
```

Comment:
I think we need version_added here even though its in the RETURN block

Reply:
exactly!

Reply:
Added.